packages/wrangler/src/index.tsx

```diff
@@ -768,7 +768,7 @@ export async function main(argv: string[]): Promise<void> {
         });
     },
     () => {
-      // "🔬 [DEPRECATED] Preview your code temporarily on cloudflareworkers.com"
+      // "🔬 [DEPRECATED] Preview your code temporarily on https://cloudflareworkers.com"
```

Comment:
This was a message from wrangler1, not meant to be clicked, but ok.

Reply:
If it doesn't matter either way, I would keep the change that makes the spellchecker happy.